src/external_recovery/external_recovery.cairo

```diff
@@ -26,12 +26,14 @@ mod external_recovery_component {
     use openzeppelin::security::reentrancyguard::{ReentrancyGuardComponent, ReentrancyGuardComponent::InternalImpl};
     use starknet::{
         get_block_timestamp, get_contract_address, get_caller_address, ContractAddress, account::Call,
-        contract_address::contract_address_const
     };
     use super::{IExternalRecoveryCallback, get_escape_call_hash};
 
     /// Minimum time for the escape security period
     const MIN_ESCAPE_PERIOD: u64 = 60 * 10; // 10 minutes;
+    
+    /// Zero address constant
+    const ZERO_ADDRESS: felt252 = 0;
```

Comment:
Here, I would do this: `pub const ZERO_ADDRESS: ContractAddress = 0.try_into().unwrap();`
And then, you can assert like `assert(guardian != ZERO_ADDRESS)` which is more readable.